docs_src/design/adr/system-management/001-docker-image-guidelines.md

```diff
@@ -0,0 +1,34 @@
+# Docker image guidelines
+
+## Status
+
+Pending
+
+## Context
+
+When deploying the EdgeX Docker containers some security measures are recommended to ensure the integrity of the software stack.
+
+## Decision
+
+When deploying Docker images, the following flags should be set for heightened security.
+
+- To avoid escalation of privileges each docker container should use the `--security-opt=no-new-privileges` flag. More details about this flag can be found [here](https://docs.docker.com/engine/reference/run/#security-configuration)
+
+- To further prevent privilege escalation attacks the user should be set for the docker container using the `--user=<userid>` or `-u=<userid>` flag. More details about this flag can be found [here](https://docs.docker.com/engine/reference/run/#user)
```

Comment:
EdgeX does not launch containers via docker run CLI.  Rewrite to focus on what is available in the docker-compose. 

Please also provide guidance on what user ID's will be used in the docker-compose.